cmd/wallet-web/src/pages/DIDAuth.vue

```diff
@@ -4,71 +4,104 @@
  * SPDX-License-Identifier: Apache-2.0
 -->
 <template>
-  <div v-if="loading" class="w-screen" style="margin-left: 40%; margin-top: 20%; height: 200px">
-    <div class="md-layout">
-      <md-progress-spinner
-        :md-diameter="100"
-        class="md-accent"
-        :md-stroke="10"
-        md-mode="indeterminate"
-      ></md-progress-spinner>
-    </div>
+  <div v-if="loading" class="chapi-container">
+    <Spinner v-if="loading" />
   </div>
 
-  <div v-else class="flex justify-center w-screen md-layout">
-    <div class="max-w-screen-md md-layout-item">
-      <md-card class="md-card-plain">
-        <md-card-header>
-          <h4 class="title">Authenticate Your Wallet</h4>
-        </md-card-header>
+  <div v-else class="flex justify-center w-screen">
+    <div>
+      <Header />
 
-        <md-card-content style="background-color: white">
-          <div v-if="errors.length">
-            <b>Failed with following error(s):</b>
-            <ul>
-              <li v-for="error in errors" :key="error">{{ error }}</li>
-            </ul>
-          </div>
+      <div class="h-auto bg-gray-light rounded-b border border-neutrals-black chapi-container">
+        <div v-if="errors.length">
+          <b>Failed with following error(s):</b>
+          <ul>
+            <li v-for="error in errors" :key="error" class="text-primary-valencia">{{ error }}</li>
+          </ul>
+        </div>
+        <!-- todo issue-1055 Read meta data from external urls -->
+        <div class="p-5 text-neutrals-dark font-sm">
+          U.S. Citizenship and Immigration Services wants to connect to your wallet.
```

Comment:
This is a generic page, so hardcoding a specific issuer name would confuse the user. Can we add a generic name like 'Issuer' rather than specific name till we fix the issue ?